package.json

```diff
@@ -38,7 +38,7 @@
     "body-parser": "1.18.3",
```

Comment:
CNR entry corresponding to this PR should be added

(Not fully sure, as there isn't any change in the component code, only tests. Although something about the full support of NGSIv2 should be included, maybe...)

Reply:
Fixed in f7dd805